mautic/mautic
# Fix duplicate entry for key campaign_rotation

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features"
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | yes/no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create campaign 1 - with landing page visits decision and modify contacts tags
3. Run this campaign for contact
4. Then create campain 2 - with same logic like campaign 1
5. Remove contact from campaign 1
6. Add contact to camapign 1 and campaign 2
7. Now when you come to page, you expect both campaigns are triggered
8. Without PR return error:
 
![image](https://user-images.githubusercontent.com/462477/137682297-3c462786-37d4-4137-8222-17e3e198c0b6.png)
